### ISSUE TITLE
Create KB article for alt text

### ISSUE THREAD

**Issue by laflannery** [department-of-veterans-affairs/va.gov-cms#10654]
## Background
In order to assist editors in creating better, more meaningful alt text for their images, we need to provide them with more information on how to do this and why they should be doing this.  A tactical and informative KB article is one way that we can provide this guidance.

## Acceptance Criteria

- [x] Article is created using the new KB article template #10653 
- [x] Article is reviewed by Sitewide CMS
- [x] Article posted to KB

**Comment by laflannery** [comment]
Proposed article is [here](https://docs.google.com/document/d/1TMy7jKfXUo-p4oQMSZGeOPqd2Ln43XQ1bRo3pnqTzxE/edit?usp=sharing). I left it in docs for ease of editing/commenting.  I have a link out to the VA content guidelines for alt text - which is still in review. Ideally I wouldn't post this until that is approved and the link exists.

@EWashb would you have an interest in reviewing this? Seeing if you think I'm missing things, didn't add enough or added too much?

**Comment by EWashb** [comment]
I think this is great @laflannery. I really enjoyed the examples section. It is good to go from my perspective.

**Comment by laflannery** [comment]
Thank you! I will keep an eye on my other article and post this when that is up.  If it won't be for a while, I will edit to remove the link and post without for now.

**Comment by laflannery** [comment]
I posted the [article](https://prod.cms.va.gov/help/cms-basics/alternative-text).  I removed my link to the VA content guidelines for now - instead I pulled out pieces and will just edit this when the other articles is published.
